modules/microshift-oc-adm-by-example-content.adoc

```diff
@@ -18,101 +16,181 @@ Mirror an operator-registry catalog
+  
+  # Collect debugging data for all clusteroperators and clusterversions
+  oc adm inspect clusteroperators,clusterversions
+----
 
-  # Collect debugging data for services
-   oc adm inspect service
 
-  # Collect debugging data for all clusterversions
-  oc adm inspect service,crd
-----
 
 == oc adm migrate icsp
-Update imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s).
+Update imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s)
 
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # update the imagecontentsourcepolicy.yaml to new imagedigestmirrorset file under directory mydir
+  # Update the imagecontentsourcepolicy.yaml file to a new imagedigestmirrorset file under the mydir directory
   oc adm migrate icsp imagecontentsourcepolicy.yaml --dest-dir mydir
 ----
 
-//== oc adm migrate template-instances
-//== oc adm must-gather
-//== oc adm new-project
 
 
-//== oc adm node-logs
-Display and filter node logs
+== oc adm ocp-certificates monitor-certificates
+Watch platform certificates.
 
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # Show kubelet logs from all masters
-  oc adm node-logs --role master -u kubelet
+  # Watch platform certificates.
+  oc adm ocp-certificates monitor-certificates
+----
+
 
-  # See what logs are available in masters in /var/logs
-  oc adm node-logs --role master --path=/
 
-  # Display cron log file from all masters
-  oc adm node-logs --role master --path=cron
+== oc adm ocp-certificates regenerate-leaf
+Regenerate client and serving certificates of an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known leaf certificates on the cluster.
+  oc adm certificates regenerate-leaf -A --all
+  
+  # Regenerate a leaf certificate contained in a particular secret.
+  oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
+----
+
+
+
+== oc adm ocp-certificates regenerate-machine-config-server-serving-cert
+Regenerate the machine config operator certificates in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate the MCO certs without modifying user-data secrets
+  oc adm certificates regenerate-machine-config-server-serving-cert --update-ignition=false
+  
+  # Update the user-data secrets to use new MCS certs
+  oc adm certificates update-ignition-ca-bundle-for-machine-config-server
+----
+
+
+
+== oc adm ocp-certificates regenerate-top-level
+Regenerate the top level certificates in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known signing certificates on the cluster.
+  oc adm certificates regenerate-signers -A --all
+  
+  # Regenerate the signing certificate contained in a particular secret.
+  oc adm certificates regenerate-signers -n openshift-kube-apiserver-operator loadbalancer-serving-signer
+----
+
+
+
+== oc adm ocp-certificates remove-old-trust
```

Comment:
Remove: This doesn't error, but I'm not seeing any change in configmaps, safe to remove from our docs for now